### PR TITLE
Updating README under test directory with a list of test scripts that perform SSH

### PR DESCRIPTION
**Pull request by veerendra-simha-garikipati**
Signed-off-by: Veerendra Garikipati <veerendra.garikipati@seagate.com>

# Problem Statement
- Updating README under test directory with a list of test scripts that perform SSH to verify functionalities.

# Design
-  Add a disclaimer in README under test directory with a list of test scripts that perform SSH to verify py-utils functionalities.

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: Y
-  Confirm All CODACY errors are resolved [Y/N]: Y

# Testing 
- [ ] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: N
- [ ] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: N
- [ ] Confirm Testing was performed with installed RPM [Y/N]:  N

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [ ] Is there a change in filename/package/module or signature [Y/N]: N
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]
- [ ] Jira is updated
- [x] Check if the description is clear and explained. 
- [x] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

-----
[View rendered py-utils/test/README.md](https://github.com/veerendra-simha-garikipati/cortx-utils/blob/test_readme_update/py-utils/test/README.md)

### CODE REVIEW

**Review by sachinpunadikar — APPROVED** [review]
+1

File: py-utils/test/README.md [hunk 1]
```diff
@@ -21,8 +21,15 @@ please email opensource@seagate.com or cortx-questions@seagate.com.
 
 ## Prerequisite for Test-Rpm build
 
-1. cortx-py-utils must be installed, if not follow [Link](https://github.com/Seagate/cortx-utils/blob/main/py-utils/README.md "cortx-py-utils installation").
-2. Kafka service up and running, if not follow [Link](https://github.com/Seagate/cortx-utils/wiki/Kafka-Server-Setup "Kafka installation")
+1. Install cortx-py-utils
+
+> Follow [cortx-py-utils installtion guide](https://github.com/Seagate/cortx-utils/blob/main/py-utils/README.md "cortx-py-utils installation")
+
+2.  Kafka and cortx-py-utils mini-provisioning
```
Comment (rahul-tripathi-git) [inline py-utils/test/README.md:28]: You may remove Kafka as it is a prerequisite for cortx-py-utils mini provisioning.

**Review by rahul-tripathi-git — APPROVED** [review]
Remove kafka from the prerequisite(since it is covered in mini provisioning pre-requisites), else provide a link for kafka installation.
Rest all looks good to me.

File: py-utils/test/README.md (hunk 1 above)
Comment (veerendra-simha-garikipati) [reply py-utils/test/README.md]: Done.

File: py-utils/test/README.md [hunk 2]
```diff
@@ -97,6 +104,8 @@ git clone https://github.com/Seagate/cortx-utils.git
 python3 -m unittest discover
 ```
 
+**Note:** Some of the py-utils unittests are dependent on other cortx components, expect some test failures when py-utils is deployed individually
```
Comment (sachinpunadikar) [inline py-utils/test/README.md:107]: reword this as below:
**Note:** Some of the py-utils unittests are dependent on other cortx components. If cortx-py-utils is deployed alone, then below test cases will not succeed:
<test cases>

Comment (veerendra-simha-garikipati) [reply py-utils/test/README.md]: Done.
